simplepso/pso.py

```diff
@@ -193,12 +165,12 @@ def update_connected(self):
         :return:
         """
         for part in self.population:
-            if part.best is None or part.best.fitness < part.fitness:
-                part.best = creator.Particle(part)
-                part.best.fitness.values = part.fitness.values
-            if self.best is None or self.best.fitness < part.fitness:
-                self.best = creator.Particle(part)
-                self.best.fitness.values = part.fitness.values
+            if part.best is None or part.best.fitness > part.fitness:
```

Comment:
I am not sure if this a fix or not, but here you reverse the inequality symbol, just a heads up